template/web/package.json

```diff
@@ -20,12 +20,14 @@
     "@sanity/block-content-to-react": "^2.0.7",
     "@sanity/image-url": "^0.140.22",
     "date-fns": "^2.19.0",
-    "gatsby": "^3.0.3",
+    "gatsby": "^4.3.0",
+    "gatsby-cli": "^4.1.0",
     "gatsby-image": "^2.0.41",
-    "gatsby-plugin-image": "^1.0.0",
-    "gatsby-plugin-postcss": "^4.0.0",
-    "gatsby-plugin-react-helmet": "^4.0.0",
-    "gatsby-source-sanity": "^7.0.2",
+    "gatsby-plugin-gatsby-cloud": "^4.2.0",
```

Comment:
I think we're not including GPGC in starters anymore so that we always install the most recent one because we install it automatically.